### PR TITLE
Adding new 2021, 2022, 2024 terms for ARC

### PR DESCRIPTION
**Pull request by justinkadi**
Added new key variables
- Unique Ship Days
- Ship Type
- Ship Flag
- Ship Size
- Black Carbon Emissions
- Lake Ice On
- Lake Ice Off
- Ice Duration
- Species Abundance
- Species Richness
- Species Distribution
- Relative Abundance
- Beaver Pond Density
- River Discharge Rate
- Ambient Ocean Noise
- Glacier and Permafrost Hazards
- Carbon Flux
- Ice Seal Vital Rates
- Mean Sea Level Pressure

Added new Other Indicators and Frostbites terms
- Satellite Record of Ship Traffic
- Lake Ice
- Arctic Geese
- Arctic Pollinators
- Beaver Engineering
- River Discharge
- Glacier and Permafrost Hazards
- Western Arctic Caribou
- Arctic Carbon Cycling
- Ice Seals

### CODE REVIEW

File: arctic-report-card/ARCRC.ttl [hunk 1]
```diff
@@ -766,6 +936,152 @@ odo:ARCRC_00000068 rdf:type owl:Class ;
...
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000076
+odo:ARCRC_00000076 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Arctic insects are sensitive indicators of ecological responses to climate change." ;
+                   obo:IAO_0000119 "Arctic Pollinators as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/arctic-pollinators/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Arctic Pollinators"@en .
```
Comment (mbjones) [inline arctic-report-card/ARCRC.ttl:1022]: Should it be "Arctic Pollinators Indicator" like the others?

File: arctic-report-card/ARCRC.ttl [hunk 2]
```diff
@@ -766,6 +936,152 @@ odo:ARCRC_00000068 rdf:type owl:Class ;
...
+###  http://purl.dataone.org/odo/ARCRC_00000072
+odo:ARCRC_00000072 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "The atmospheric pressure at mean sea level, either directly measured or, most commonly, empirically determined from the observed station pressure." ;
+                   obo:IAO_0000119 "mean sea level pressure as defined at https://glossary.ametsoc.org/wiki/Sea_level_pressure" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Vital Sign, 'Surface Air Temperature'" ;
+                   rdfs:label "mean sea level pressure"@en ;
+                   skos:narrowMatch "http://purl.dataone.org/odo/ECSO_00001650" .
```
Comment (mbjones) [inline arctic-report-card/ARCRC.ttl:982]: I really like it when we can make these linkages to existing terms. Can you describe why this isn't an exact match?

File: arctic-report-card/ARCRC.ttl [hunk 3]
```diff
@@ -766,6 +936,152 @@ odo:ARCRC_00000068 rdf:type owl:Class ;
...
+###  http://purl.dataone.org/odo/ARCRC_00000070
+odo:ARCRC_00000070 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "A carbon flux refers to the amount of carbon exchanged between carbon stocks over a specified time. In simple terms, it is the movement of carbon between land, oceans, atmosphere, and living things." ;
+                   obo:IAO_0000119 "carbon balance as defined at https://www.canr.msu.edu/news/carbon-fluxes-and-carbon-stocks#" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Arctic Carbon Cycling'" ;
+                   rdfs:label "carbon flux"@en ;
+                   skos:equivalentClass "http://purl.dataone.org/odo/ECSO_00001105" .
```
Comment (mbjones) [inline arctic-report-card/ARCRC.ttl:959]: Excellent.

File: arctic-report-card/ARCRC.ttl [hunk 4]
```diff
@@ -766,6 +936,152 @@ odo:ARCRC_00000068 rdf:type owl:Class ;
...
+
+###  http://purl.dataone.org/odo/ARCRC_00000071
+odo:ARCRC_00000071 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Data for each seal included collection date, species, sex, standard length, and blubber thickness." ;
+                   obo:IAO_0000119 "ice seal vital rates as defined in the Arctic Report Card 2024 Essay Ice Seals" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Ice Seals'" ;
+                   rdfs:label "ice seal vital rates"@en .
```
Comment (mbjones) [inline arctic-report-card/ARCRC.ttl:970]: Why are the vital rates lumped together as one key variable, when multiple things really seem to be being measured? Should these be broken apart? I think this is worth a discussion with Sandy and team at some point to try to resolve just what a Key Variable really is.

File: arctic-report-card/ARCRC.ttl [hunk 5]
```diff
@@ -679,7 +848,8 @@ odo:ARCRC_00000061 rdf:type owl:Class ;
 ###  http://purl.dataone.org/odo/ARCRC_00000062
 odo:ARCRC_00000062 rdf:type owl:Class ;
                    rdfs:subClassOf odo:ARCRC_00000040 ;
-                   obo:IAO_0000115 "Difference in seasonal or annual air temperatures from the 1981-2010 mean (a positive anomaly is a higher temperature; a negative anomaly is a lower temperature)." ;
+                   obo:IAO_0000115 "Difference in seasonal or annual air temperatures from a reference value or long-term average (a positive anomaly is a higher temperature; a negative anomaly is a lower temperature)." ;
+                   obo:IAO_0000119 "seasonal/annual anomaly patterns for air temperature as defined in https://www.ncei.noaa.gov/access/monitoring/global-temperature-anomalies" ;
```
Comment (mbjones) [inline arctic-report-card/ARCRC.ttl:852]: Why did you change this definition? Was the previous definition wrong? Do they circumscribe the same set of members, and, if not, shouldn't we define a new class? We should discuss this one.

File: arctic-report-card/ARCRC.ttl [hunk 6]
```diff
@@ -470,6 +470,163 @@ odo:ARCRC_00000025 rdf:type owl:Class ;
...
+
+###  http://purl.dataone.org/odo/ARCRC_00000029
+odo:ARCRC_00000029 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "measurement of maximum tonnage of ships" ;
+                   obo:IAO_0000119 "ship size as defined in the Arctic Report Card 2022 Essay at https://arctic.noaa.gov/report-card/report-card-2022/satellite-record-of-pan-arctic-maritime-ship-traffic/" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-11-21T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Satellite Record of Pan-Arctic Maritime Ship Traffic'" ;
+                   rdfs:label "ship size"@en .
```
Comment (mbjones) [inline arctic-report-card/ARCRC.ttl:514]: Can we link these to existing ontology terms? There are a lot of ship/ocean vessel vocabularies I think. e.g., see the vocabs in use at BCO-DMO.

**Review by mbjones — COMMENTED** [review]
In general this looks like a nice, tidy update and expansion. I'd like to see terms related to existing vocabularies whenever possible. Also, I left a few notes and questions in the the code below for us to discuss. Let's discuss and then you can cut a release after any changes get made.

File: arctic-report-card/ARCRC.ttl (hunk 1 above)
Comment (justinkadi) [reply arctic-report-card/ARCRC.ttl]: Since these were "Frostbite" essays, the other year's Frostbite essays didn't have indicators next to them, so I wrote it like this to match those. But I think changing them to have "Indicator" at the end would be good. I've changed it to Arctic Pollinators Indicator and Glacier & Permafrost Hazards Indicator. I left the previous year's "Western Alaska Salmon" and "Working with Communities to Observe the Arctic" unchanged though.

File: arctic-report-card/ARCRC.ttl (hunk 2 above)
Comment (justinkadi) [reply arctic-report-card/ARCRC.ttl]: This was in reference to mean sea level pressure. You're right, it should be an exact match. I thought "mean sea level pressure" was more narrow than "sea level pressure", but I see that the definition for sea level pressure mentions it's the average pressure

File: arctic-report-card/ARCRC.ttl (hunk 4 above)
Comment (justinkadi) [reply arctic-report-card/ARCRC.ttl:975]: This author hadn't actually provided any key variables, so I came up with ice seal vital signs. I can ask them for clarification and maybe provide it in an update after the report card has been published next week.

File: arctic-report-card/ARCRC.ttl (hunk 5 above)
Comment (justinkadi) [reply arctic-report-card/ARCRC.ttl:857]: I decided to change this definition because I felt the original definition was too specific since it mentioned "Difference in seasonal or annual air temperatures from the 1981-2010 mean". I changed it to "a reference value or long-term average".

File: arctic-report-card/ARCRC.ttl (hunk 6 above)
Comment (justinkadi) [reply arctic-report-card/ARCRC.ttl]: Added narrowMatch and relatedMatches to the ship key variables

**Comment by justinkadi** [comment]
I added some more links, edited some label names, and added 2 new terms that an Author just added today (burned area and fire carbon emissions)
